AsyncDisplayKit/Details/ASRangeController.mm

```diff
@@ -129,11 +132,41 @@ - (void)addNodeToWorkingRange:(ASCellNode *)node
 - (void)moveNode:(ASCellNode *)node toView:(UIView *)view
 {
   ASDisplayNodeAssertMainThread();
-  ASDisplayNodeAssert(node && view, @"invalid argument, did you mean -removeNodeFromWorkingRange:?");
+  ASDisplayNodeAssert(node && view, @"invalid argument, did you mean -removeNodeFromRenderRange:?");
 
   [view addSubview:node.view];
 }
 
+- (void)addNode:(ASCellNode *)node toRange:(ASLayoutRange)range
+{
+  switch (range) {
```

Comment:
nit: I would like to register an action and performSelector to generalize and simplify this part.
Anyway, it is nit